addToCart/index.html

```diff
@@ -0,0 +1,28 @@
+<!<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Add to cart</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    
+</head>
+<body>
+    <div class='wrapper'>
+        <div class='completed'>
+            <svg class='check_mark' viewBox='0 0 24 24'>
```

Comment:
let's use the naming of classes and ids with the dash and not with underscore from now on
ex: `check_mark` -> `check-mark`

Reply:
Done